src/dosbox.cpp

```diff
@@ -1239,6 +1239,13 @@ void DOSBOX_Init()
 	        "tab-separated format, used by SETVER.EXE as a persistent storage\n"
 	        "(empty by default).");
 
+	pbool = secprop->Add_bool("file_locking", only_at_start, true);
+	pbool->Set_help(
+	        "Enable file locking (SHARE.EXE emulation).\n"
+	        "This is required for some Windows 3.1 applications to work properly.\n"
+	        "It generally does not cause problems for DOS games except in rare cases (Astral Blur demo).\n"
+	        "If you experience crashes related to file permssions, you can try disabling this (enabled by default).");
+
```

Comment:
Also, starting from the second line, could you wrap the rest manually to 80 chars? We've done this for all the other setting descriptions.

Reply:
fixed